niworkflows/func/util.py

```diff
@@ -251,9 +252,12 @@ def init_enhance_and_skullstrip_bold_wf(
                         run_without_submitting=True)
 
     # Run N4 normally, force num_threads=1 for stability (images are small, no need for >1)
-    n4_correct = pe.Node(ants.N4BiasFieldCorrection(
-        dimension=3, copy_header=True, bspline_fitting_distance=200),
+    n4_correct = pe.Node(N4BiasFieldCorrection(
+        dimension=3, copy_header=True, bspline_fitting_distance=200), shrink_factor=2,
         name='n4_correct', n_procs=1)
+    _n4_version = N4BiasFieldCorrection().version
+    if _n4_version and parseversion(_n4_version) >= Version('2.1.0'):
+        n4_correct.inputs.rescale_intensities = True
```

Comment:
Again. Is there an advantage to accommodating ANTs versions that don't do what we need?

Reply:
I see it a bit differently here, I totally agree with you.